python/tank/deploy/util.py

```diff
@@ -38,6 +38,39 @@ def execute_git_command(cmd):
     if status != 0:
         raise TankError("Error executing git operation. The git command '%s' returned error code %s." % (cmd, status))     
 
+def execute_toolkit_command(pipeline_config_path, toolkit_command, args):
```

Comment:
@manneohrstrom This was to stay consistent with [process_manager](https://github.com/shotgunsoftware/tk-framework-desktopserver/blob/b7b2936bba261371e8c21b1209ca54e99e0d745b/python/tk_framework_desktopserver/process_manager.py#L170).
Do you think we should use `execute_tank_command` anyway?